src/app/public/plugin-resources/code-examples/country-field/basic/country-field-demo.component.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  Component,
+  OnInit
+} from '@angular/core';
+
+import {
+  FormControl,
+  FormGroup,
+  Validators
+} from '@angular/forms';
+
+import {
+  SkyCountryFieldCountry
+} from '@skyux/lookup';
+
+@Component({
+  selector: 'app-country-field-demo',
+  templateUrl: './country-field-demo.component.html'
+})
+export class CountryFieldDemoComponent implements OnInit {
+
+  public countryData: SkyCountryFieldCountry;
+
+  public countryForm: FormGroup;
+
+  public countryControl: FormControl;
```

Comment:
Check your alphabetical ordering of properties here ☝️

Reply:
Done